### ISSUE TITLE
Squeezed image on pdf - 1px height bug

### ISSUE THREAD

**Issue by Hibron** [DavBfr/dart_pdf#274]
On some JPG images class PdfJpegInfo calculates wrong 1px height. In a result image is squeezed to 1px height in PDF file.
Image is from original camera app from Samsung Galaxy Tab A, Android 8.1

![image](https://user-images.githubusercontent.com/18200004/79895095-a61fe600-8406-11ea-8866-0868dc265f99.jpg)

Use above image in:
PdfImage.jpeg or PdfImage.file

then it will result in image squeezed to 1px in PDF. 
Result PDF:
[document-wrong.pdf](https://github.com/DavBfr/dart_pdf/files/4511687/document-wrong.pdf)
You can see PDF is 2MB and image is almost invisible because squeezed to 1px.

Code to reproduce issue(use my image posted above):

```
image = pdfLib.PdfImage.jpegSized(
        pdf.document,
        image: (File(IMAGE_PATH).readAsBytesSync()),
      );

pdf.addPage(pw.Page(build: (pw.Context context) {
        return pw.Center(
          child: pw.Image(image),
        ); // Center
      }
```

Height of above image is calculated in line 59 in exif.dart:
`height = buffer.getUint16(offset + 1);`
It reads height of this image as 1.

I made workaround and changed image.dart in my fork: 
https://github.com/Hibron/dart_pdf/blob/master/pdf/lib/src/image.dart
I added method PdfImage.jpegSized . Now when I use PdfImage.jpegSized instead of PdfImage.jpeg or PdfImage.file , I have correctly sized photo on my pdf.

Resolution:
I used JpegDecoder().startDecode() method to obtain width and height for image instead of PdfJpegInfo.

**Comment by DavBfr** [comment]
Thanks, I'll take a look.

**Comment by DavBfr** [comment]
It's fixed in version 1.6.2.
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
